### ISSUE TITLE
design system, components [COGNITION]: Screen reader announcement MUST match text on screen, pagination

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/vets-design-system-documentation#374]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

On the first results page, the `<h2>` will read "Showing 1-10...", however, when the user clicks on "Next" or "2" and focus is brought back to the newly updated string "Showing 11-20..."-- the _prior_ page's string is announced "Showing 1-10".

<img width="678" alt="Screen Shot 2021-01-12 at 7 08 42 PM" src="https://user-images.githubusercontent.com/14154792/104389537-b2565100-5509-11eb-99f8-2cba19b21cee.png">

Also, NVDA will not announce "Showing 1-10..." when focus is moved there after searching. It will only announce that results are loading.

<img width="677" alt="Screen Shot 2021-01-12 at 7 12 14 PM" src="https://user-images.githubusercontent.com/14154792/104389736-2264d700-550a-11eb-8f1d-d2519494a00a.png">

## What I expected to happen

- [ ] I expect NVDA to announce the current string.
- [ ] Create and/or modify unit tests for this component to include an aXe scan

## Reproducing
- Formation version:
- Device: Windows
- Browser: NVDA / Firefox

Steps to reproduce:

1. Go to `https://staging.va.gov/find-forms/?q=medical` on Firefox with NVDA
2. Navigate to the second page
3. Confirm the announced information is not the same as the text on screen

**Comment by zacharymorel** [comment]
@joshkimux Note, this user story is under Decision Tools umbrella. Sorry of the confusion.

**Comment by noahgelman** [comment]
Closed as this was fixed previously
